code/modules/heavy_vehicle/equipment/utility.dm

```diff
@@ -621,7 +621,7 @@
 
 /obj/item/mecha_equipment/autolathe/afterattack(atom/target, mob/living/user, inrange, params)
 	. = ..()
-	if(istype(target, /obj/item/stack/material/steel) || istype(target, /obj/item/stack/material/glass))
+	if(istype(target, /obj/item/stack/material/steel) || istype(target, /obj/item/stack/material/glass) || istype(target, /obj/item/stack/material/aluminium) || istype(target, /obj/item/stack/material/lead) || istype(target, /obj/item/stack/material/plastic))
```

Comment:
Use `is_type_in_list` instead of 30 istype OR'd

Reply:
should be better now, thanks for telling me